### ISSUE TITLE
 [pywebview] GTK cannot be loaded

### ISSUE THREAD

**Issue by mpsingh47** [r0x0r/pywebview#1241]
### Specification
- pywebview version: 4.3.3
- operating system: Ubuntu 22.04
- web renderer: dont know

### Description

script.py >>>
import webview
window = webview.create_window('Woah dude!', 'https://pywebview.flowrl.com')
webview.start()

ERROR: when i do > python script.py
 [pywebview] GTK cannot be loaded
Traceback (most recent call last):
  File "/home/manpreetsingh7/venv/leap/lib/python3.10/site-packages/webview/guilib.py", line 25, in import_gtk
    import webview.platforms.gtk as guilib
  File "/home/manpreetsingh7/venv/leap/lib/python3.10/site-packages/webview/platforms/gtk.py", line 20, in <module>
    import gi
ModuleNotFoundError: No module named 'gi'
QSocketNotifier: Can only be used with threads started with QThread

### Practicalities

- YES/NO I am willing to work on this issue myself.

- YES/NO I am prepared to support this issue financially.

**Comment by starnight** [comment]
I think #1234 is a good reference, including the dependencies.  :)

**Comment by r0x0r** [comment]
you need to install gtk dependencies. see https://pywebview.flowrl.com/guide/installation.html#dependencies

**Comment by 17hoehbr** [comment]
I am running into a similar error, but for some reason it only occurs when I run pyinstaller in Github actions. If I run my app directly from the python file or from a pyinstaller binary built locally it works fine.

Here is the error I am getting: 
```
[pywebview] GTK cannot be loaded
Traceback (most recent call last):
  File "webview/platforms/gtk.py", line 26, in <module>
  File "gi/__init__.py", line 126, in require_version
ValueError: Namespace WebKit2 not available
```

And [here](https://github.com/17hoehbr/OpenMic-Karaoke/blob/main/.github/workflows/build-appimage.yml) is the Github action I have setup:
```
name: Build AppImage

on:
  push:
    branches:
      - main
    paths:
      - 'app.py'
      - 'build.spec'
      - 'OpenMic.desktop'
      - 'requirements.txt'
      - 'templates/**'
      - 'static/**'
      - '.github/workflows/build-appimage.yml'

jobs:
  build:
    runs-on: ubuntu-20.04

    steps:
    - name: Checkout code
      uses: actions/checkout@v3

    - name: Set up Python
      uses: actions/setup-python@v3
      with:
        python-version: 3.12

    - name: Install dependencies
      run: |
        sudo apt-get update
        sudo apt-get install -y libglib2.0-0 libgtk-3-0 libwebkit2gtk-4.0-dev libfuse2 appstream libgirepository1.0-dev python3-gi python3-gi-cairo gobject-introspection gir1.2-gtk-3.0 gir1.2-webkit2-4.0
    - name: Install build tools
      run: sudo apt-get install -y build-essential

    - name: Install PyInstaller
      run: |
        pip install pyinstaller
        pip install pywebview[gtk]
        pip install -r requirements.txt

    - name: Build executable
      run: pyinstaller build.spec  # Replace "your_script.py" with the entry point of your Python application

    - name: Create AppDir
      run: |
        mkdir -p AppDir
        cp dist/OpenMic AppDir/
        cp OpenMic.desktop AppDir/
        cp static/icons/logo.png AppDir/openmic.png

    - name: Download linuxdeploy
      run: |
        wget -O linuxdeploy https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage
        chmod +x linuxdeploy

    - name: Create AppImage
      run: |
        ./linuxdeploy --appdir=AppDir --output appimage --desktop-file AppDir/OpenMic.desktop --icon-file AppDir/openmic.png --executable AppDir/OpenMic

    - name: Upload AppImage as artifact
      uses: actions/upload-artifact@v3
      with:
        name: OpenMic
        path: OpenMic-x86_64.AppImage
```

Running these same steps creates a working AppImage on my local Fedora install. I suspect that there is a missing package but I've tried everything in the documentation to no avail.

**Comment by starnight** [comment]
Does this commit (["[GTK] Fix WebKit 4.0 as fallback"](https://github.com/r0x0r/pywebview/commit/f2a63a2d197eb30a9fb913d1b5fe39dfa0c80f5d)) help?
